rest-api-spec/src/main/resources/rest-api-spec/api/exists_source.json

```diff
@@ -3,8 +3,8 @@
     "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/docs-get.html",
     "methods": ["HEAD"],
     "url": {
-      "path": "/{index}/{type}/{id}/_source",
-      "paths": ["/{index}/{type}/{id}/_source"],
+      "path": "/{index}/{id}/_source",
```

Comment:
In our proposal, we actually decided the new endpoint should be `/{index}/_source/{id}`. Here was our reasoning for not simply removing the type component (as in `{index}/{id}/_source`):
- Unlike types, IDs are allowed to contain underscores, and there could be collisions with other endpoints like `{index}/_doc/{id}`.
- It doesn’t work cleanly for all URLs, for example 'termvectors' where it introduces collisions between the different endpoints.